### ISSUE TITLE
Replace antrun with xolstice/protobuf-maven-plugin

### ISSUE THREAD

**Issue by jcoeltjen** [btc-ag/service-idl#39]
Consider using the [xolstice/protobuf-maven-plugin](https://github.com/xolstice/protobuf-maven-plugin) instead of the antrun plugin as this results in way cleaner pom files.

Also make sure the .proto files will be placed in the **src/main/proto** folder.

**Comment by sigiesec** [comment]
Yes, I also think that plugin should be used, I already added a comment in the code here: https://github.com/btc-ag/service-idl/blob/b464cf1e61530eb4cd10b4a18593c60ab45f273f/com.btc.serviceidl/src/com/btc/serviceidl/generator/java/POMGenerator.xtend#L25

**Comment by sigiesec** [comment]
> Also make sure the .proto files will be placed in the src/main/proto folder.

Hm, why do you mention this here? At least in the example I have seen, the .proto files are already placed in that folder.

**Comment by jcoeltjen** [comment]
> Hm, why do you mention this here? At least in the example I have seen, the .proto files are already placed in that folder.

I was not sure about that at the moment I wrote the issue. If it is already the case.. even better.

**Comment by sigiesec** [comment]
Ok :)
